samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -60,33 +60,49 @@ nRF52 core output
 
 .. code-block:: console
 
-   *** Power Management Demo on nrf52_pca10040 ***
-   Demo Description
-   Application creates Idleness, Due to which System Idle Thread is
-   scheduled and it enters into various Low Power States.
+  *** Power Management Demo on nrf52_pca10040 ***
+  Demo Description
+  Application creates idleness, due to which System Idle Thread is
+  scheduled and it enters into various Low Power States.
 
-   <-- App doing busy wait for 10 Sec -->
+  <-- App doing busy wait for 10 Sec -->
 
-   <-- App going to sleep for 10 Sec -->
-   --> Entering to SYS_POWER_STATE_CPU_LPS_1 state.
-   --> Exited from SYS_POWER_STATE_CPU_LPS_1 state.
+  <-- App going to sleep for 10 Sec -->
+  --> Entering to SYS_POWER_STATE_CPU_LPS_1 state.
```

Comment:
Please enclose in double backticks ` `` ` anything that is code or variables:
` ``SYS_POWER_STATE_CPU_LPS_1`` `

Reply:
In the docs of in the example? This is log of the example execution.

Reply:
Sorry, my mistake

Reply:
Right, this is in a code-block so is OK as is.